### DISCUSSION TITLE
Jupyter book vs MyST-md for an engineering report: almost there but some issues

### DISCUSSION THREAD

**Discussion by giorgio-caumo** [executablebooks/meta#1210] [Q&A]
Hi everyone,
I hope you can help me with some issues I am encountering with both Jupyter book and MyST-md.
I have looked at some of the previous questions in the forum but haven't found yet something similar to what I specifically ask, but feel free to reference previous discussions.
In the README I go more in detail about my questions. 
I also uploaded all the possible files necessary to recreate my issues and possibly solutions, as well I may need specific codes to obtain what I need.

Since it is my first time on GitHub I am still getting familiar with how it works.
I created a repository with the files, you can find it here: [https://github.com/giorgio-caumo/executable-notebook.git](https://github.com/giorgio-caumo/executable-notebook.git)
And I also upload the repository as a zip.
[executable-notebook-main.zip](https://github.com/user-attachments/files/16599259/executable-notebook-main.zip)
If there are some issues with the files, please let me know, I'll try to upload them again.

In short I would want to create an automated engineering report, with specific template and specific layout, by embedding markdowns (more 'reporting' part) and jupyter notebooks (more 'calculation' part). 
Right now I am obtaining good results in the html based part with both, but I need to produce a static pdf.

Hope you can help me,
Thank you
Giorgio
